### PR TITLE
Workflows GeoTIFF Validator update

### PR DESCRIPTION
**Pull request by hblitza**
This updates the example workflows to test the `geotiff-validator` worker.

@JakobMiksch please check.

### CODE REVIEW

File: workflows/publish-geotiff-with-validator-error.json [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "id": 3,
+      "type": "geoserver-publish-geotiff",
+      "inputs": [
+        "klips",
```
Comment (JakobMiksch) [inline workflows/publish-geotiff-with-validator-error.json:32]:
```suggestion
        "dresden",
```

the klips workspace does not exist by default I think

File: workflows/publish-geotiff-with-validator.json [hunk 2]
```diff
@@ -1,37 +1,70 @@
...
+            "maxFileSize": 10000000
+          }
+        }
+      ]
+    },
+    {
+      "id": 3,
+      "type": "geoserver-publish-geotiff",
+      "inputs": [
+        "klips",
```
Comment (JakobMiksch) [inline workflows/publish-geotiff-with-validator.json:58]:
```suggestion
        "langenfeld",
```

the klips workspace does not exist by default I think

**Review by JakobMiksch — APPROVED** [review]
can be merged, after https://github.com/klips-project/klips-api/pull/41 and https://github.com/klips-project/rabbitmq-worker/pull/118 are merged

**Comment by hblitza** [comment]
thx

**Pull request merged**